packages/ove-service-imagetiles/src/OVE.Service.ImageTiles/Domain/ImageProcessingStates.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace OVE.Service.ImageTiles.Domain {
 
-    public enum ProcessingStates {
+    public enum ImageProcessingStates {

```

Comment:
The processing states below are not the ones defined on the `appsettings.json`. Can this be fixed?

Reply:
Following a code-first approach the states in the json file are replaced via reflection from those in https://github.com/ove/ove-asset-services/blob/Archives/packages/ove-service-archives/src/OVE.Service.Archives/Domain/ArchiveProcessingStates.cs so this is only here as indication to the user. Will reduce to -1 error and 0 unprocessed which are standard across all services.